lib/core/versions/latest/DocumentComposer.ts

```diff
@@ -21,15 +21,47 @@ export default class DocumentComposer {
       return { status: 'deactivated' };
     }
 
+    const document = didState.document as DocumentModel;
+
+    // Only transform `publicKeys` to  `publickey` if the array is present.
+    let publicKeys;
+    if (Array.isArray(document.publicKeys)) {
+      publicKeys = [];
+      for (let publicKey of document.publicKeys) {
+        const didDocumentPublicKey = {
+          id: '#' + publicKey.id,
+          controller: '',
+          type: publicKey.type,
+          publicKeyJwk: publicKey.jwk
```

Comment:
This is the line that converts the key for external representation.

Reply:
Yes.